Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -504,20 +540,41 @@ def fetch_incidents(client: Client, last_run: Dict[str, int],
                         attachments.append(tmp_attachment)
 
         alert["attachments"] = attachments
-        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
-        if alert_csv_id:
-            extracted_csv_data = extract_data_from_csv_stream(client, alert_id,  # type: ignore
-                                                              alert_csv_id)
-            alert['alert_data']['csv'] = extracted_csv_data
+        alert_data = dict_safe_get(alert, ['alert_data', 'csv'])

```

Comment:
```suggestion
        alert_data = dict_safe_get(alert, ['alert_data', 'csv'], {})
```
We can take advantage of "default_return_value" argument of dict_safe_get function.